classes/genre.rb

```diff
@@ -0,0 +1,10 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name)
```

Comment:
Please, kindly add 'id' to the initialize method. It should take 'id' parameter as well.

Reply:
@zhorabay The Id and item are private attributes

Reply:
You can check the UML diagram for reference